api/app/signals/apps/api/serializers/nested/reporter.py

```diff
@@ -39,3 +45,10 @@ def to_internal_value(self, data):
             data['phone'] = None
 
         return super().to_internal_value(data)
+
+    def get_allows_contact(self, instance) -> bool:
```

Comment:
This class method can be deleted when using `allows_contact = serializers.BooleanField(source='_signal.allows_contact', read_only=True)`